docs/api.md

```diff
@@ -1202,6 +1208,12 @@ for the authenticated user.
   
   <!--end-response-body-get-accountsessions-isCurrentDevice-->
 
+* `isMemoryToken`: *boolean, required*
```

Comment:
With appologies for the late drive-by comment, this seems like it's piercing a layer of abstraction - it's taking an internal implementation detail of the server and exposing it in the API for consumption by the client.  To display the correct information, the content-server needs to know that the auth-server sometimes stores tokens in redis, and that these tokens are the only ones that are up-to-date.

I wonder if a better abstraction here would involve us sending two separate timestamps:

* `lastAccessTime`: the value from redis, but it might be `null` if we don't have a correct value to show
* `createdTime`: the "started syncing" timestamp which we know we can always get from the db